docs/src/api-testing-python.md

```diff
@@ -0,0 +1,289 @@
+---
+id: api-testing
+title: "API testing"
+---
+
+Playwright can be used to get access to the [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) API of
+your application.
+
+Sometimes you may want to send requests to the server directly from Node.js without loading a page and running js code in it.
+A few examples where it may come in handy:
+- Test your server API.
+- Prepare server side state before visiting the web application in a test.
+- Validate server side post-conditions after running some actions in the browser.
+
+All of that could be achieved via [APIRequestContext] methods.
+
+The following examples rely on the [`pytest-playwright`](./test-runners.md) package which add Playwright fixtures to the Pytest test-runner.
+
+<!-- TOC -->
+
+## Writing API Test
+
+[APIRequestContext] can send all kinds of HTTP(S) requests over network.
+
+The following example demonstrates how to use Playwright to test issues creation via [GitHub API](https://docs.github.com/en/rest). The test suite will do the following:
+- Create a new repository before running tests.
+- Create a few issues and validate server state.
+- Delete the repository after running tests.
+
+### Configure
+
+GitHub API requires authorization, so we'll configure the token once for all tests. While at it, we'll also set the `baseURL` to simplify the tests.
+
+```python
```

Comment:
Shouldn't it be `python sync ` here and below?

Reply:
The Playwright Pytest plugin is only available in sync.